cluster-autoscaler/integration/framework.go

```diff
@@ -163,6 +163,125 @@ func (driver *Driver) runAutoscaler() {
 	Expect(autoscalerSession.ExitCode()).Should(Equal(-1))
 }
 
+func getStorageClassObject(class string) (*v1storage.StorageClass, error) {
+	provider := os.Getenv("cluster_provider")
```

Comment:
update the `env_variable` to the new one.

Reply:
Env variable is still cluster_provider. This is going to get exported after `make download-kubeconfigs` same as the SEED name entered during the above make.

Reply:
Changes are done according to our discussion.